benchmark_runner/common/oc/oc.py

```diff
@@ -48,6 +49,104 @@ def get_ocp_server_version(self):
         """
         return self.run(f"{self.__cli} get clusterversion version -o jsonpath='{{.status.desired.version}}'")
 
+    def upgrade_ocp(self, upgrade_ocp_version: str):
+        """
+        This method upgrades OCP version with conditional handling for specific versions.
+
+        @param upgrade_ocp_version: Version to upgrade to
+        @return:
+        """
+        ocp_channel = '.'.join(upgrade_ocp_version.split('.')[:2])
+        upgrade_command = f"{self.__cli} adm upgrade ; {self.__cli} adm upgrade channel stable-{ocp_channel} ; {self.__cli} adm upgrade --to={upgrade_ocp_version} --allow-explicit-upgrade;"
+
+        if ocp_channel == "4.16":
+            patch_command = f"{self.__cli} -n openshift-config patch cm admin-acks --patch '{{\"data\":{{\"ack-4.15-kube-1.29-api-removals-in-4.16\":\"true\"}}}}' --type=merge"
+            self.run(patch_command)
+
+        # sometimes upgrade is not working on the first run
```

Comment:
What goes wrong the first time?

Reply:
Seems that the upgrade did not start

Reply:
Change the comment to:
# The upgrade is triggered only when it is run twice